openai/chatgpt-template/web/routes/chat.tsx

```diff
@@ -3,9 +3,10 @@ import { useUser } from "@gadgetinc/react";
 import CurrentChat from "../components/CurrentChat";
 import LeftNav from "../components/LeftNav";
 import { api } from "../api";
+import { User } from "@gadget-client/chatgpt-template";
 
 export default function () {
-  const user = useUser(api);
+  const user = useUser(api) as any as User;
```

Comment:
This seems like a bit of a red flag - our useUser hook should return a User type without casing to `any`

Reply:
This is a bug that Harry and I talked about